cargo/.github/workflows/rust_ci.yml

```diff
@@ -0,0 +1,51 @@
+name: Continuous Integration
+
+on:
+  push:
+    paths-ignore:
+      - "**/README.md"
+  pull_request:
+  workflow_dispatch:
+
+env:
+  CARGO_TERM_COLOR: always
+  {% raw -%}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  {%- endraw %}
+
+jobs:
+  rust-checks:
+    name: Rust Checks
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        action:
+          - command: build
+            args: --release
+          - command: fmt
+            args: --all -- --check --color always
+          - command: clippy
+            args: --all-targets --all-features --workspace -- -D warnings
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Enable caching
+        uses: Swatinem/rust-cache@v2
+      - name: Setup Rust
+      {%- if arch == "riscv" %}
+        uses: dtolnay/rust-toolchain@v1
+        with:
+          toolchain: nightly
+          components: rust-src
+      {%- else %}
+        uses: esp-rs/xtensa-toolchain@v1.5
+        with:
+          default: true
+          buildtargets: {{ mcu }}
+          ldproxy: false
+      {%- endif %}
+      - name: Run command
+        {%- raw %}
+        run: cargo ${{ matrix.action.command }} ${{ matrix.action.args }}
```

Comment:
Hmmm... where are these coming from?

Reply:
You mean the [`raw` tags](https://shopify.dev/docs/api/liquid/tags/raw)? If so, its to avoid cargo-generate parsing the GH actions variables like  `${{ matrix.action.command }}` since they use the same syntax (`{{ <var_name> }}`)

Reply:
No I mean I do not see `matrix.action.command` and `matrix.action.args` defined anywhere, even if with a default value.
Is the user supposed to always type their content by himself? Would they know what to enter there?

Reply:
Those are defined in the matrix, here: https://github.com/esp-rs/esp-idf-template/blob/cea1382543faf56322ffd3507b9617adea70a57e/cargo/.github/workflows/rust_ci.yml#L23-L29